javascripts/govuk/show-hide-content.js

```diff
@@ -0,0 +1,150 @@
+(function (global) {
+  'use strict';
+
+  var $ = global.jQuery;
+  var GOVUK = global.GOVUK || {};
+
+  function ShowHideContent() {
+    var self = this;
+
+    // Radio and Checkbox selectors
+    var selectors = {
+      namespace: 'ShowHideContent',
+      radio: '.block-label input[type="radio"]',
+      checkbox: '.block-label input[type="checkbox"]'
```

Comment:
Hi @nickcolley, don't forget this isn't my module. I've only refactored, tested and migrated it here.

If you want to change it's behaviour or styling let's open another pull request. It's already in use elsewhere using this class so we'd have to think about backwards compatibility.